### PR TITLE
Cert and SET cmdlets bugfix.

### PR DESCRIPTION
**Pull request by kagamsft**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@kagamsft__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (kaga). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by stankovski** [comment]
Started MSI build: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-msi/369/
Started on-demand build: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-on-demand/39/

**Comment by kagamsft** [comment]
Build is fine. Could someone please merge my PR.

**Comment by stankovski** [comment]
@kagamsft please review output from msi job

**Comment by kagamsft** [comment]
Build failed because of signing. Here is the error:

JobCompletionPath: \\PR1PD1CDSCVLT04.NORTHAMERICA.CORP.MICROSOFT.COM\02\CMP\FXSIGN\PRA4728927
JobNumber: PRA4728927
IsSuccess: False
Watching Job results
Job is finished, Success=False  Assemblies Signed=127  BytesSigned=28741120
Code Signing Failure:Code Signing Failure {Code Signing Failure}
:This file does not appear to be "delay signed".  For more information please visit http://codesigninfo/Wiki/StrongName.aspx.: Release-ResourceManager-AzureResourceManager-SiteRecovery-Microsoft.Azure.Commands.SiteRecovery.dll {This file does not appear to be "delay signed".  For more information please visit http://codesigninfo/Wiki/StrongName.aspx.: Release-ResourceManager-AzureResourceManager-SiteRecovery-Microsoft.Azure.Commands.SiteRecovery.dll}
Signing failed
csu.exe Error: 0 : Signing failed
Copying signed files from \\adxsdkbuilder\\Signed to D:/workspace/powershell-msi\src\Package\Release ...
0
Verify that all assemblies are signed ...
Cleaning up \\adxsdkbuilder\\ToSign ...
Deleting signed files from \\adxsdkbuilder\\Signed ...
C:\ci-signing\adxsdk\AzurePowerShell\SignAssemblies.ps1 : Assemblies not 
successfully signed after 3 tries
At C:\Users\fxsign\AppData\Local\Temp\hudson5527209974155441290.ps1:1 char:1
+ C:\ci-signing\adxsdk\AzurePowerShell\SignAssemblies.ps1
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorExcep 
   tion
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorExceptio 
   n,SignAssemblies.ps1

**Comment by yugangw-msft** [comment]
restart msi run of http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-msi/389/, which passed

File: src/ResourceManager/Network/Commands.Network/ApplicationGateway/SslCertificate/AzureApplicationGatewaySslCertificateBase.cs [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.Network.Models;
```
Comment (yugangw-msft) [inline src/ResourceManager/Network/Commands.Network/ApplicationGateway/SslCertificate/AzureApplicationGatewaySslCertificateBase.cs]: Next time, sort the namespaces

**Pull request merged**